readthedocs/restapi/templates/restapi/log.txt

```diff
@@ -0,0 +1,13 @@
+RTD build information
+Build: {{ build.id }}
+Project: {{ build.project }}
+Version: {{ build.commit }}
```

Comment:
This is a commit, not a version. We have a thing that is already called a Version, which is different.